src/components/dataset/entity/list.vue

```diff
@@ -0,0 +1,142 @@
+<!--
```

Comment:
all of these components nested in dataset/entities could move up to an entities folder